components/NutrientTable/NutrientTable.tsx

```diff
@@ -0,0 +1,25 @@
+import type { Nutrient } from '../../types/product';
+
+type NutrientTableProps = {
+  nutrition: Nutrient[];
+};
+
+export const NutrientTable = ({ nutrition }: NutrientTableProps) => {
+  return (
+    <ul className="w-full max-w-sm mx-auto">
```

Comment:
You use `Container` component for this purpose. It should be higher in the hierarchy, maybe in the Page abstraction.
```suggestion
    <ul className="w-full">
```